Rename expert labels to topic labels (#1076)
python/devguide: triage/labels.rst

  (written in Python) and other changes related to tests, :mod:`unittest`,
  or :mod:`doctest`.

OS labels
=========

These labels are used to specify which operating systems are affected.
Since most issues either affect all systems or are specific to Unix,
the only available labels are :gh-label:`OS-windows`, :gh-label:`OS-mac`,
and :gh-label:`OS-freebsd`.

.. _Expert labels:

Topic labels

> ============

Reviewer: Is this heading going to get the automatic `Topic labels` reST target and `topic-labels` HTML ID and an extra target `Expert labels` for compatibility, or is the explicit markup going to override the automatic things?

Author: We get these HTML anchor IDs:

* https://cpython-devguide--1076.org.readthedocs.build/triage/labels/#expert-labels
* https://cpython-devguide--1076.org.readthedocs.build/triage/labels/#topic-labels

And we get just the reST target to to `Expert labels`.

The main reason for adding `.. _Expert labels:` is to make sure the old HTML anchor still works.

But shall we also add a `Topic labels` reST target, like this?

```diff
diff --git a/triage/labels.rst b/triage/labels.rst

 .. _Expert labels:
+.. _Topic labels:
 
 Topic labels
 ============
```

Reviewer: Ideally the reST target would be `Topic labels`.  It that requires explicit markup without losing the other ID, then let’s do that.  (If it doesn’t work try switching the two lines.)

Author: Updated, so now we have HTML anchors and reST targets for both!

Reviewer 2: The ref target label, if needed, should be the standard `.. _topic-labels:` , as used for all other modern ref target labels in the devguide. While there's no immediate benefit to adding such if the target isn't used, it makes it easier for any new inbounds to add such, so I generally favor doing so.

  (written in Python) and other changes related to tests, :mod:`unittest`,
  or :mod:`doctest`.

OS labels
=========

These labels are used to specify which operating systems are affected.
Since most issues either affect all systems or are specific to Unix,
the only available labels are :gh-label:`OS-windows`, :gh-label:`OS-mac`,
and :gh-label:`OS-freebsd`.

.. _Expert labels:

Topic labels
============

> These labels are used to specify the area of expertise required to address

Reviewer 2: Maybe worth updating this a bit too, to reflect the title change and avoid implying that being an expert is required to interact with the PR?

```suggestion
These labels are used to denote the specific topic area, if any, of
```
